### ISSUE TITLE
Wildcard for excludePaths

### ISSUE THREAD

**Issue by icanhazstring** [goaop/goaop-zf2-module#2]
The `excludePaths` currently on uses a simple array with multiple paths.
Would be nice to be able to provide a wildcard path which gets checked if a path matches a wild card.

[Console/Warmup/Zf2WarmupCommond](https://github.com/goaop/goaop-zf2-module/blob/1.1.0/src/Console/Warmup/Zf2WarmupCommand.php#L67)
```php
$enumerator = new Enumerator($options['appDir'], $options['includePaths'], $options['excludePaths']);
$iterator   = $enumerator->enumerate();
```

This way is would be possible to exclude configs from classes, e.g.:
```php
'excludePaths' => [
   $dir . '/module/**/config'
]
```

The core problem is the [Enumerator from goaop/framework](https://github.com/goaop/framework/blob/2.0.0/src/Instrument/FileSystem/Enumerator.php#L108)

```php
foreach ($excludePaths as $excludePath) {
    if (strpos($realPath, $excludePath) === 0) {
        return false;
    }
}
```

@lisachenko maybe this ticket belongs to goaop/framework. but i don't know. Your choice, i would fix this :)

**Comment by lisachenko** [comment]
Yes, agree. Your idea can be useful for several cases

**Comment by icanhazstring** [comment]
--Edit-- Updated ticket to be more precise on the problem

**Comment by lisachenko** [comment]
@icanhazstring maybe we can replace `strpos` with `fnmatch` and this will be ok? But need to test path separators, dots and other symbols carefully

**Comment by icanhazstring** [comment]
@lisachenko will try to provide some tests then and create the fix for goaop/framework

**Comment by icanhazstring** [comment]
I created pull request in goaop/framework

https://github.com/goaop/framework/pull/300
Tests provided, this should resolve this request.

**Comment by icanhazstring** [comment]
Closed by goaop/framework#300
